client/app/components/user/session.controller.js

```diff
@@ -1,45 +1,32 @@
 angular.module("App").controller("SessionCtrl", [
     "$scope",
+    "$document",
     "translator",
-    "User",
+    "SessionService",
 
-    function ($scope, translator, User) {
+    function ($scope, $document, translator, SessionService) {
         "use strict";
 
-        $scope.accountInfo = {
-            activated: false
-        };
-        $scope.loader = {
-            accountInfo: false
-        };
-
-        $scope.selectedLanguage = translator.getSelectedAvailableLanguage();
-
-        // languages choice
-        $scope.availableLanguages = translator.getAvailableLanguages();
-        $scope.$watch("selectedLanguage", (newLanguage) => {
-            $scope.updateSelectedLanguage(newLanguage);
-        });
-
         $scope.$watch("i18n.global_app_title", () => {
             translator.setTitle($scope.tr("global_app_title"));
         });
 
-        $scope.updateSelectedLanguage = function (newLanguage) {
-            $scope.selectedLanguage = newLanguage;
-            translator.setLanguage(newLanguage.value);
-            translator.setTitle($scope.tr("global_app_title"));
-        };
-
-        User.getV3Url().then((url) => {
-            $scope.v3url = url;
-        });
-
         // FIX for /me/alerts
         $scope.$on("Navigator.navigationInformationsChange", (e, data) => {
             $scope.isLeftMenuVisible = data && data.leftMenuVisible;
         });
 
-        translator.setTitle($scope.tr("global_app_title"));
+        // Scroll to anchor id
+        $scope.scrollTo = (id) => {
+            // Set focus to target
+            $document.find(`#${id}`)[0].focus();
```

Comment:
Maybe you should check if the `id` exists before using the element returned by `find` but I don't know that happens when the `id` isn't found

Reply:
It's only used once for the moment. But yes, we could. 
Updated